src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -12,35 +12,35 @@
     "origin.toomanyorigins.undead.name": "Undead",
 
     "origin.toomanyorigins.withered.name": "Withered",
-    "origin.toomanyorigins.withered.description": "Witheredlar, atalarının Wither'ın kaotik gücünden yararlanması sonucu öldürdüklerinin yaşam enerjisini emer.",
+    "origin.toomanyorigins.withered.description": "Atalarının Wither'ın kaotik gücünden yararlanması sonucu öldürdüklerinin yaşam enerjisini emerler.",
   
-    "power.toomanyorigins.overheat.name": "Aşırı Isınma",
-    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni incitir.",
+    "power.toomanyorigins.workup.name": "Workup",
+    "power.toomanyorigins.workup.description": "Baktığın herhangi bir nesneden dışa doğru roket zıplayışı gerçekleştirebilirsin, bunu beklemeden kullanmak sana hasar verir.",  
     "power.toomanyorigins.aerial_stability.name": "Hava Dengesi",
-    "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızınız biraz azaltılmış.",
+    "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızın biraz azaltılmış.",
```

Comment:
changed "siz(multiple or formal form of you)" to "sen(singular form of you)" here